src/Reflectivity/MetaLinkNodesMapper.class.st

```diff
@@ -34,8 +34,8 @@ MetaLinkNodesMapper >> findNodesForObject: anObject [
 
 { #category : #initialize }
 MetaLinkNodesMapper >> initialize [
-	objectsForNodes := WeakKeyDictionary new.
-	nodesForObjects := WeakKeyDictionary new
+	objectsForNodes := WeakIdentityKeyDictionary new.
+	nodesForObjects := WeakIdentityKeyDictionary new
```

Comment:
Do you have a test which shows that weak part is really working?
I did little example in workspace and links are not collected for me when I close everything.

Reply:
Indeed there is a problem. It is a separate issue, I just opened https://github.com/pharo-project/pharo/issues/6575